tsk/vs/tsk_gpt.h

```diff
@@ -28,6 +28,18 @@ extern "C" {
 #define GPT_HEAD_OFFSET	1
 #define GPT_HEAD_SIG	0x5452415020494645ULL
 
+/*Length of a GUID partition type description*/
+#define GUID_DESC_LEN 64
+
+/*Definiation of GUID struct.*/
+    typedef struct{
+        uint32_t data_1;
+        uint16_t data_2;
+        uint16_t data_3;
+        uint8_t data_4[8];
+    } GUID;
```

Comment:
This should have a GPT-specific name.  To be consistent with other volume systems, something like gpt_guid.

Reply:
@bcarrier - in fact, it *is* a GPT-specific name. These are the names that Microsoft uses in its documentation!
https://learn.microsoft.com/en-us/windows/win32/api/guiddef/ns-guiddef-guid